Formula/openssl@3.rb

```diff
@@ -52,6 +53,7 @@ def configure_args
       --prefix=#{prefix}
       --openssldir=#{openssldir}
       --libdir=#{lib}
+      enable-fips
```

Comment:
Is there a reason this isn't enabled by default? We're usually a bit leery of enabling features that aren't the default since upstream often knows better about what's sensible.

Do any other package managers enable this?

Reply:
This pull request https://github.com/openssl/openssl/pull/13684, which was accepted shortly before openssl 3.0.0, disabled the fips module by default and reorganized the installation instructions a bit. The discussion of the PR alludes to some of the subtleties involved: If you upgrade the fips module, you need to retest it, but maybe you don't want to do that at that point. So there needs to be some extra care and alarm applied in some cases. At least that's how I understand it. I don't think these kinds of nuances apply to a Homebrew installation.

I see that Fedora has the fips module enabled in their openssl 3 packages. Of course, the Red Hat operating systems have a long history of deep integration of fips mode.

Reply:
If upstream disables it by default because it seems too complicated for users I'm not sure Homebrew should be re-enabling it by default.

Reply:
> I don't think these kinds of nuances apply to a Homebrew installation.

Why do you think that?